resources/frontend-development.md

```diff
@@ -1,14 +1,70 @@
 # Frontend Development
 
+Frontend Development is a pivotal aspect of web development that focuses on crafting the visual and interactive components of a web application. It encompasses two fundamental elements: the graphical design and the user interface (UI) development.
+
+## Graphical Design
+
+In the context of frontend development, graphical design involves conceptualizing and creating the visually appealing layouts, typography,color schemes, and graphical elements that collectively form the aesthetic foundation of the application.Designers employ tools like Adobe Photoshop, Sketch, or Figma to translate ideas into tangible visual assets.
```

Comment:
Just some small type changes for readability here:

`In the context of frontend development, graphical design involves conceptualizing and creating the visually appealing layouts, typography, color schemes, and graphical elements that collectively form the aesthetic foundation of the application. Designers employ tools like Adobe Photoshop, Sketch, or Figma to translate ideas into tangible visual assets.`

Reply:
Done. Thank you for the review, @mariahlaqua.